samples/embedded-map/src/components/EmbeddedMap/EmbeddedMapModel.ts

```diff
@@ -103,29 +172,45 @@ export default class EmbeddedMapModel extends ComponentModelBase {
             return;
         }
 
+        await whenDefinedOnce(this.map.view, "center");
+
+        // Set mapillary to this location
+        await this._moveCloseToPosition(
+            this.map.view.center.latitude,
+            this.map.view.center.longitude
+        );
+
         // Create location marker based on current location from Mapillary and
         // pan/zoom Geocortex map to the location.
-        const [{ lat, lon }, bearing] = await Promise.all([
-            this.mapillary.getPosition(),
-            this.mapillary.getBearing(),
-        ]);
-        const centerPoint = new Point({ latitude: lat, longitude: lon });
+        const {
+            latitude,
+            longitude,
+            heading,
+            tilt,
+            fov,
+        } = await this._getMapillaryCamera();
+
+        const centerPoint = new Point({ latitude, longitude });
         await Promise.all([
             this.messages.commands.locationMarker.create.execute({
-                fov: 30,
+                fov,
                 geometry: centerPoint,
-                heading: bearing,
+                heading,
+                tilt,
                 id: this.id,
                 maps: this.map,
-            }),
-            this.messages.commands.map.zoomToViewpoint.execute({
-                maps: this.map,
-                viewpoint: {
-                    rotation: getCameraRotationFromBearing(bearing),
-                    targetGeometry: centerPoint,
-                    scale: 3000,
-                },
-            }),
+                userDraggable: true,
+            } as any),
```

Comment:
Why is this `any` cast necessary?

Reply:
To get it to build using 5.8. I think I must be missing something as this is the latest version I can see when running npm info @vertigis/web

Reply:
Similar to the comment by Eric above, we can use an `rc` version of the package and change to `5.9.0` to get the correct typings before we merge.

Reply:
If 5.8 is the latest thing you see for `npm info @vertigis/web`, then likely you don't have a .npmrc file set up to point at our internal package registry. Safest thing is to put it in your home directory, or one level up from the clone of this repo so it doesn't get committed by accident. Ping me in Teams on Monday and I can set you up if you're unsure (don't really want to put instructions here for all the world to see).

Reply:
5.9.0 is now published to npm. You can update the dependency in package.json.

Reply:
Looks like everything is building correctly now